Makefile

```
@@ -176,6 +176,42 @@ build-mcuboot: submodules zephyr fprime-venv
 
+##@ Debugging / OpenOCD
+
+OPENOCD_DIR ?= $(shell pwd)/tools/openocd
+OPENOCD_REPO ?= https://github.com/raspberrypi/openocd.git
+OPENOCD_BIN ?= $(OPENOCD_DIR)/src/openocd
+OPENOCD_JOBS ?= 4
+OPENOCD_FLASH_SPEED ?= 5000
+OPENOCD_COMMON_FLAGS ?= -s tcl -f interface/cmsis-dap.cfg -f target/rp2350.cfg -c "adapter speed $(OPENOCD_FLASH_SPEED)"
+
+.PHONY: openocd
+openocd: ## Download and build a local OpenOCD binary from the Raspberry Pi fork
```

checkmake (0.3.2)

[warning] Target body for "openocd" exceeds allowed length of 5 lines (7).

(maxbodylength)